tests/sentry/api/endpoints/test_event_committers.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+from django.core.urlresolvers import reverse
+
+from sentry.testutils import APITestCase
+
+
+class EventCommittersTest(APITestCase):
+    def test_simple(self):
```

Comment:
ok fair, I got lazy because of the amount of objects it needs to populate but it's not that bad and we'll need them eventually anyway